### ISSUE TITLE
CMS data tables missing scope attributes

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10130]
## Description
Multiple data tables are missing the scope attribute on the `<th>` cells which may make it difficult for screen readers to read the table data correctly.

## Additional Context
List of tables where this was found (may not be the full list of all tables within the CMS):

- Event Edit screen: Date & Time Table
- Health Services List Edit screen: Featured Content
- Leadership List Edit screen: Leadership
- VAMC Facility Edit screen: Location Services Table
- VAMC System Billing and Insurance Edit screen: Hours Table
- VAMC System Operating Status Edit screen: System-wide alerts Table
- VAMC System Operating System Edit screen: Facility Operating statuses Table

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183909646-62e693a6-f063-4ff4-9e3a-fb7d8d478b1e.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Technical review for adding scope to tables (one update for all or individual updates)
- [ ] Change management consulted
- [ ] Implementation ticket(s) created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
THis should be a one and done situation

**Comment by srancour** [comment]
This is something that should be done and relates to my [comments](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17755#issuecomment-2073551085) on #17718

**Comment by gracekretschmer-metrostar** [comment]
Scope of impact: a user with a screen reader are missing context to related to table headers.

**Comment by JakeBapple** [comment]
Adding some template overrides, I have added in scope="col" attribute and replaced empty <th> elements with <td> elements per WCAG standards. This fix also includes style updates to make sure the updated <td> elements look like former <th> elements.

- [x] Event Edit screen: Date & Time Table
- [x] Health Services List Edit screen: Featured Content
- [ ] Leadership List Edit screen: Leadership
- [ ] VAMC Facility Edit screen: Location Services Table
- [ ] VAMC System Billing and Insurance Edit screen: Hours Table
- [ ] VAMC System Operating Status Edit screen: System-wide alerts Table
- [ ] VAMC System Operating System Edit screen: Facility Operating statuses Table
- [ ] Vet Center - Locations List Edit screen: Display additional mobile vet centers Table

will keep updating this comment instead of leaving it open.
